heatcheck/backend/queries/comments.js

```diff
@@ -0,0 +1,60 @@
+const db = require('../db/index');
+
+const getAllComments = async (req, res, next) => {
+    try {
+        let { post_id } = req.params
+        let comments = await db.any("SELECT * FROM comments WHERE post_id = $1", post_id)
+        res.status(200).json({
+            comments,
+            status: "success",
+            message: "all comments for post"
+        })
+    } catch (error) {
+        next (error)
+    }
+}
+
+const addSingleComment =  async (req, res, next) => {
+    try {
+        let { post_id } = req.params
+        let { user_id, body } = req.body
+        await db.none("INSERT INTO comments (user_id, post_id, body) VALUES ($1, $2, $3)", [user_id, post_id, body])
```

Comment:
This should be db.one and add RETURNING * to the query to get back the new comment as a part of the response